swarm/tools/runs_gc.py

```
@@ -74,13 +75,26 @@ def age_days(self) -> float:
     total = 0
+    # Performance optimization: use os.scandir instead of Path.rglob to avoid
+    # overhead from object creation and separate stat() calls.
     try:
-        for entry in path.rglob("*"):
-            if entry.is_file():
-                try:
-                    total += entry.stat().st_size
-                except OSError:
-                    pass
+
+        def scan_dir(dir_path: Path):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `scan_dir`

Add return type annotation: `None`

(ANN202)